<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="renderButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>581</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Render</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Canvas Size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="canvasWidthLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="canvasHeightLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Light</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lightPositionLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Intensity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lightIntensityLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fov</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cameraPositionLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cameraFovLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sphere</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="spherePositionLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sphereRadiusLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sphereColorLineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addSpherePushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Sphere</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Triangle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="triangleColorLineEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="trianglePosALineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position b</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="trianglePosBLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addTrianglePushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Triangle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position a</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="trianglePosCLineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Position c</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>-10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ray Tracer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
